backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -1,169 +1,102 @@
-// Navigation
-//---------------------------------------------------
-.inline-menu {
-  margin: 0;
-  -webkit-margin-before: 0;
-  -webkit-padding-start: 0;
+.admin-nav {
+  @include display(flex);
+  @include flex-direction(column);
+  border-right: 1px solid $color-sidebar-border;
+  background: $color-sidebar-bg;
 }
 
-nav.menu {
-  ul {
-    list-style: none;
-
-    li {
-      a {
-        padding: 10px 0;
-        display: block;
-        position: relative;
-        text-align: left;
-        border: 1px solid transparent;
-        text-transform: uppercase;
-        font-weight: $font-weight-bold;
-        font-size: 90%;
-      }
-
-      &.active a {
```

Comment:
Since this is not the global `nav.menu` style anymore, we should keep these styles and rename the class to `nav.content-sub-menu`. But this means we need to change the html class of the content sub menu, which could break overrides. 

So, the best would be to keep them as `nav.menu` IMO, although this naming is misleading.